crates/core/src/mock/fixtures/responses/starknet_call.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Responses need to follow the enum for `JsonRpcResponse` from starknet-rs, meaning they can't contain a field "jsonrpc".